gps/solver.go

```diff
@@ -105,12 +105,9 @@ type SolveParameters struct {
 	// typical case.
 	Downgrade bool
 
-	// Trace controls whether the solver will generate informative trace output
-	// as it moves through the solving process.
-	Trace bool
```

Comment:
Let's just leave everything underneath gps/ out of this refactor, unless it absolutely _must_ change.